values.yaml

```diff
@@ -67,6 +67,7 @@ persistence:
   enabled: false
   size: 10Gi
   # storageClass: '-'
+  deleteEnabled: false
```

Comment:
Missing documentation for `persistence.deleteEnabled` in **README.md**. At first look, this seems potentially spurious, but I can see how it may be necessary for garbage collection to function.

Reply:
potentially spurious indeed. maybe @kuzaxak added it simply because it was [missing](https://github.com/twuni/docker-registry.helm/search?q=deleteenabled) from values.yaml? 

potentially similar to the other spurious comments above

Reply:
```suggestion
```